src/PKSim.Infrastructure/Services/PKSimLogger.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+using OSPSuite.Core.Domain;
+using ILogger = OSPSuite.Core.Services.ILogger;
+
+namespace PKSim.Infrastructure.Services
+{
+   public class PKSimLogger : ILogger
```

Comment:
Interface ILogger (from us this time) should probably change. We will take care of that once 7.2 is released and we can work in 7.3 OSPSuite,Core etc.